contracts/voting/cwd-voting-cw721-staked/src/contract.rs

```diff
@@ -45,6 +45,8 @@ pub fn instantiate(
 
     TOTAL_STAKED_NFTS.save(deps.storage, &Uint128::zero(), env.block.height)?;
 
+    DAO_ADDRESS.save(deps.storage, &info.sender)?;
```

Comment:
how about moving this above the `let owner = // ...` expression so that we can avoid the clone?

Reply:
Done.